src/components/user-profile.html

```diff
@@ -1,125 +1,142 @@
+            <td><jha-icon-phone></jha-icon-phone></td>
+            <td>[[user.phoneNumber]]</td>
           </tr>
-          <template is="dom-if" if="[[isExpanded]]">
-            <tr>
-              <td>Phone:</td>
-              <td>[[user.phoneNumber]]</td>
-            </tr>
-            <tr>
-              <td>Email:</td>
-              <td>[[user.email]]</td>
-            </tr>
-          </template>
           <tr>
-            <td><button on-click="cardExpandCollapse">[[collapseButtonText(isExpanded)]]</button></td>
-            <td><button type="button" on-click="editUserProfile">[[editButtonText(editSelected)]]</button></td>
+            <td><jha-icon-email></jha-icon-email></td>
+            <td>[[user.email]]</td>
           </tr>
-        </table>
+        </template>
+      </table>
+      <button on-click="cardExpandCollapse">[[collapseButtonText(isExpanded)]]</button>
+      <button type="button" on-click="editUserProfile">[[editButtonText(editSelected)]]<jha-icon-edit style="height: 16px; width: 16px; vertical-align: text-bottom; padding-left: 5px; fill:#fcfcff;"></jha-icon-edit></button>
+      <button class="delete" type="button"  on-click="deleteUserProfile" indexed$=[[user.id]]>Delete <jha-icon-delete style="height: 16px; width: 16px; vertical-align: text-bottom; fill:#fcfcff;"></jha-icon-delete></button>
     </template>
     <template is="dom-if" if="[[editSelected]]">
         <table>
           <tr>
-            <td><label for="firstName">First name :</label></td>
+            <td><label for="firstName">First name </label></td>
             <td><input type="text" name="firstName" id="firstName" value="{{user.firstName}}" on-input="updateUserFirstName"></td>
           </tr>
           <tr>
-            <td><label for="lastName">Last name :</label></td>
+            <td><label for="lastName">Last name </label></td>
             <td><input type="text" name="lastName" id="lastName" value="{{user.lastName}}" on-input="updateUserLastName"></td>
           </tr>
           <tr>
-            <td><label for="phone">Phone :</label></td>
+            <td><label for="phone">Phone </label></td>
             <td><input type="text" name="phone" id="phone" value="{{user.phoneNumber}}" on-input="updateUserPhoneNumber"></td>
           </tr>
           <tr>
-            <td><label for="email">Email :</label></td>
+            <td><label for="email">Email </label></td>
             <td><input type="text" name="email" id="email" value="{{user.email}}" on-input="updateUserEmail"></td>
           </tr>
-          <tr>
-            <td><button type="button" on-click="editUserProfile">[[editButtonText(editSelected)]]</button></td>
-          </tr>
         </table>
+        <button type="button" on-click="editUserProfile">[[editButtonText(editSelected)]]</button>
     </template>
+    
   </template>
   <script type="module">
     'use strict';
+    import localStorageApi from "../api/local-storage.js";
```

Comment:
this should be passed down from `intern-app`, not imported directly. otherwise you're going to have at least two different `localStorageApi` instances, which could cause problems.

Reply:
not sure if I've passed data down from an import like this before do you do it the same way as passing static get properties? Like through the components `local-storage-api=[[localStorageApi]]` or does it need to be reassigned?

Reply:
> Like through the components local-storage-api=[[localStorageApi]] or does it need to be reassigned?

yes and kind of yes. You'll need to add a property in your polymer properties that's set to `localStorageApi`. But at that point you'll pass it to your components like any other defined polymer property.